core/domain/feedback_services.py

```diff
@@ -115,8 +118,16 @@ def create_message(
             thread.subject = updated_subject
     thread.put()
 
+    if (user_services.is_user_registered(author_id) and len(text) > 0 and
```

Comment:
Note: this one is still important -- we should at least know what the current behaviour is, so that we can file an issue if needed.